### DISCUSSION TITLE
Email unreachable, password lost, cant create support online

### DISCUSSION THREAD

**Discussion by green-frog** [community/community#22119] [New to GitHub]
<p>Hello community,</p>
<p>Without notice, because a new french law allow them to, my provider recently deleted the email I use on Github.<br>
So I wanted to change it in my settings for the new one.<br>
<strong>sudo mode</strong> requires me to provide my account password and, unfortunately, I dont remember it.</p>
<p>I tried to create a support with the form but I cant send it because it requires a valid email (list is empty for me and is required)</p>
<p>Anyone know how I can get someone from the support team to help me changing my email ?<br>
I’m still connected on browser and my Oauth token works (but I dont have role user:email. I would be able to use developer API to reach /user/emails endpoint if that was the case).</p>
<p>Thanks for reading me.</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>Hi there! :wave: Welcome to the Community!</p>
<p>If you use an Incognito or Private browsing window, you should still be able to get in touch using our <a href="https://support.github.com/contact">contact form</a>, even if you’re unable to login. Your issue would require direct contact with the support team.</p>
<p>When you’re logged in, if the contact form is showing you an empty dropdown for email addresses, could you share a screenshot of that and which browser you’re using? It should definitely display a list of any registered email addresses you have on your account.</p>
<p>(please do not share screenshots that reveal personal information)</p>

**Comment by green-frog** [comment]
<p>Hello !</p>
<p>Thanks for your answer.<br>
I will use the form without being authenticated ! Good advice :+1:</p>
<p>I’m using Mozilla Firefox for Ubuntu 78.0.1 (64-bit) but I don’t think this is related to browser.</p>
<p>Here is two screenshots in one file :</p>
<ul>
<li>Contact form : first <code>select</code> (<strong>From</strong>) has no <code>option</code> so there is nothing to select<br>
The second <code>select</code> (<strong>Account or organization</strong>) show all my accounts &amp; organizations.</li>
<li>Email settings of my github account : this may happen because of the <strong>Undeliverable</strong> &amp; <strong>Unverified</strong> primary email status. Therefore, it seems normal to me not to offer an invalid email in the list</li>
</ul>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/b/ba59daec4350c2ef057006d5843a0f15735f2945.png" data-download-href="/uploads/short-url/qAxk2OlU4kPQm30b8jr8jMGQipL.png?dl=1" title="github community" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/1693179/181068928-c89dd3b7-10b5-43ec-a56a-b0234e15b5f9.png" alt="github community" data-base62-sha1="qAxk2OlU4kPQm30b8jr8jMGQipL" width="690" height="360" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/b/ba59daec4350c2ef057006d5843a0f15735f2945_2_690x360.png, https://aws1.discourse-cdn.com/github/optimized/2X/b/ba59daec4350c2ef057006d5843a0f15735f2945_2_1035x540.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/b/ba59daec4350c2ef057006d5843a0f15735f2945_2_1380x720.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/b/ba59daec4350c2ef057006d5843a0f15735f2945_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github community</span><span class="informations">1490×779 89.9 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>The issue there is that I never verified my email &amp; set a backup one. Won’t happen again :sweat_smile:</p>
